tests/src/model/pages/pods-page.ts

```diff
@@ -92,4 +82,11 @@ export class PodsPage extends BasePage {
     await this.playKubernetesYAMLButton.click();
     return new PlayKubeYamlPage(this.page);
   }
+
+  async prunePods(): Promise<PodsPage> {
+    await this.prunePodsButton.click();
+    await this.pruneConfirmationButton.waitFor({ state: 'visible', timeout: 3000 });
+    await this.pruneConfirmationButton.click();
```

Comment:
also using the dialog handling would be nice.

Reply:
done